### PR TITLE
Add Resource Manager Guide

### PR DESCRIPTION
**Pull request by blowmage**
This PR adds the overview documentation as well as fixes some smaller formatting issues with the existing documentation.

### CODE REVIEW

File: lib/gcloud/resource_manager.rb [hunk 1]
```diff
@@ -61,9 +61,132 @@ def self.resource_manager keyfile = nil, options = {}
...
+  # * Update existing projects
+  # * Delete projects
+  # * Undelete, or recover, projects that you don't want to delete
+  #
+  # The Resource Manager API is a Beta release and is not covered by any SLA or
+  # deprecation policy and may be subject to backward-incompatible changes.
+  #
+  # == Accessing the Service
+  #
+  # The Resource Manager API cannot be enabled in the developer console, and can
```
Comment (jgeewax) [inline lib/gcloud/resource_manager.rb]: Maybe bold `*cannot*`

**Comment by jgeewax** [comment]
LGTM

File: lib/gcloud/resource_manager.rb (hunk 1 above)
Comment (blowmage) [inline lib/gcloud/resource_manager.rb]: Emboldened!

**Comment by quartzmo** [comment]
This should complete Resource Manager documentation, and make this service ready for review.

**Pull request merged**